scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -48,10 +48,51 @@ function populateCytobands(cytobands){
   }
 }
 
-function populateStart(){
-  startElem.value = cytoStart.options[cytoStart.selectedIndex].value
+
+
+// ValidateForm()
+// Controll user input fields (start, end) in varaint filter.
+//
+function validateForm(){
```

Comment:
👍 Yes, it's much better to have this function in here, where all types of variants can call it!